packages/types/src/atoms/graphql-types.ts

```diff
@@ -10,6 +10,7 @@ export type Scalars = {
   Int: number;
   Float: number;
   Date: string;
+  DateTime: string;
```

Comment:
Think this might cause issues, prefer DateTimeOffset for it's timezone handling

Reply:
Good spot, this'll be the line at fault. 
<img width="760" alt="image" src="https://user-images.githubusercontent.com/63285990/229783003-39b007d2-4fb3-40c2-a5ff-47cdac1e12b0.png">
Will need an ehr-client change and clearly I'm not thinking straight enough to be making lots of small PRs, that'll just put undue burden on the rest of the team. I'll come back to this later